code/game/objects/items/stacks/wrap.dm

```diff
@@ -86,7 +89,7 @@
 			I.forceMove(P)
 			var/size = round(I.w_class)
 			P.name = "[weightclass2text(size)] parcel"
-			P.w_class = size
+			P.w_class = max(I.w_class+0.3, WEIGHT_CLASS_HUGE)
```

Comment:
Nitpick, but if you did `I.w_class + (1 / 3)` it'd probably be more precise, so as to avoid decimal rounding silliness.

Reply:
/4 or /2 would be even better for code handling.